modules/metering-exposing-the-reporting-api.adoc

```diff
@@ -52,18 +54,18 @@ Be sure to replace `testuser:password123` with a valid username and password com
 
 [id="metering-manually-configure-authentication_{context}"]
 == Manually Configuring Authentication
-To manually configure, or disable OAuth in the `reporting-operator`, you must set `spec.tls.enabled: false` in your MeteringConfig.
+To manually configure, or disable OAuth in the Reporting Operator, you must set `spec.tls.enabled: false` in your `MeteringConfig` resource.
 
 [WARNING]
 ====
-This also disables all TLS/authentication between the `reporting-operator`, presto, and hive. You would need to manually configure these resources yourself.
+This also disables all TLS and authentication between the Reporting Operator, Presto, and Hive. You would need to manually configure these resources yourself.
 ====
 
-Authentication can be enabled by configuring the following options. Enabling authentication configures the `reporting-operator` pod to run the OpenShift auth-proxy as a sidecar container in the pod. This adjusts the ports so that the `reporting-operator` API isn't exposed directly, but instead is proxied to via the auth-proxy sidecar container.
+Authentication can be enabled by configuring the following options. Enabling authentication configures the Reporting Operator pod to run the OpenShift auth-proxy as a sidecar container in the pod. This adjusts the ports so that the Reporting Operator API isn't exposed directly, but instead is proxied to via the auth-proxy sidecar container.
```

Comment:
There's a couple of mentions that document this as the "reporting API" vs. the "Reporting Operator API".

Reply:
@timflannagan Do you recommend "reporting API" or "Reporting Operator API"?